parsers/thueringen.py

```diff
@@ -0,0 +1,169 @@
+from urllib.request import urlopen
+from bs4 import BeautifulSoup as parse
+import re
+from utils import Parser
+from datetime import datetime, timedelta, date
+from pyopenmensa.feed import LazyBuilder
+
+amount_regex = re.compile('\d{1,2},\d{1,2}')
+
+
+def parse_start_date(document):
+	week_start_end_date_regex = re.compile('\d{1,2}\.\d{1,2}\.\d{4}')
+	calendar_week_regex = re.compile('\d{4}-\d{1,2}$')
+
+	option = document.find('option', value=calendar_week_regex, selected=True)
+
+	start_date_str = week_start_end_date_regex.search(option.text)
+
+	return datetime.strptime(start_date_str.group(), '%d.%m.%Y').date()
+
+
+def parse_available_weeks(document):
+
+	sel_week = document.find('select', {"name": 'selWeek'})
+
+	options = sel_week.find_all('option')
+
+	for option in options:
+		week = option['value']
+		if week != '0':
+			yield week
+
+
+def parse_fees(document):
+	fees_regex = re.compile('Bedienstete.*')
+
+	fees = document.find_all('p', class_='MsoNormal', string=fees_regex)
+
+	for fee in fees:
+
+		fee_strings = fees_regex.findall(fee.text)
+		for amount_candidate in fee_strings:
+			amount_strings = amount_regex.findall(amount_candidate)
+			if len(amount_strings) != 2:
+				continue
+
+			employees_fee = float(amount_strings[0].replace(',', '.'))
+			guests_fee = float(amount_strings[1].replace(',', '.'))
+			return employees_fee, guests_fee
+
+	return None, None
+
+
+def parse_ingredients(document):
+	groups_regex = re.compile('([\w\d*]+):\s+(.*)')
+	groups = dict()
+
+	for table in document.select('div.kontextbox > table:nth-of-type(1)'):
+		for s in table.stripped_strings:
+			if groups_regex.search(s):
+				split = s.split(':')
+				groups[split[0]] = split[1]
```

Comment:
The object returned by `search` already contains the matched groups (the parenthesis in the regex), which can be accesses using `group(1)` and `group(2)`, so you don't need `split`. You should also strip the description here instead of in the list comprehension in line 84.